dispatch/modules/podcasts/models.py

```diff
@@ -84,7 +87,13 @@ def __init__(self, *args, **kwargs):
     file = FileField(upload_to='podcasts/')
 
     def get_absolute_url(self):
-        return self.file.url
+        return "%spodcast/%s#%s" % (settings.BASE_URL, self.podcast.slug, self.get_slug_from_title())
+
+    def get_file_url(self):
+        return "%s%s/" % (settings.MEDIA_URL, self.file)
+
+    def get_slug_from_title(self):
```

Comment:
I think it might be better to add an actual `slug` field to the model, even if it's auto-generated from the title on the first save (see the `Topic` mode). Once a URL is published (i.e. on Facebook), it's important to keep the URL the same to prevent dead links. In this case, the URL could change if somebody updates the title.

Alternatively, could you just use the episode number?

Reply:
sidenote django have `slugify` method
```
>>> from django.template.defaultfilters import slugify
>>> slugify("b b b b")
u'b-b-b-b'
>>>
```

Reply:
I think ill use the episode number to avoid the migration